src/libraries/Common/src/Interop/Windows/Ole32/Interop.CoGetContextToken.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Ole32
+    {
+        internal static unsafe int CoGetContextToken(out IntPtr token)
+        {
+            fixed (IntPtr* pToken = &token)
+            {
+                return CoGetContextToken(pToken);
+            }
+        }
+
+        [LibraryImport(Libraries.Ole32)]
+        internal static unsafe partial int CoGetContextToken(IntPtr* pToken);
```

Comment:
This is the same as having manually written the wrapper, so just use LibraryImport directly.

```suggestion
        [LibraryImport(Libraries.Ole32)]
        internal static unsafe partial int CoGetContextToken(out IntPtr pToken);
```